src/store/portfolio/index.js

```diff
@@ -0,0 +1,66 @@
+const initialState = {
+  data: [],
+  history: [],
+  assets: [
+    {
+      name: "Bitcoin",
+      purchase_price: "39000",
+      price_on_purchase_date: "",
+      purchase_date: "18-01-2020",
+    },
+  ],
+  selectedCoin: [],
+  purchase_price: "",
+  purchase_date: "",
+  isLoading: false,
+  hasError: false,
+  error: "",
+};
+export const SELECT_COIN = "ADD_ASSET";
+export const GET_COIN_HISTORY = "GET_COIN_HISTORY";
+export const PURCHASE_AMOUNT = "PURCHASE_AMOUNT";
+export const PURCHASE_DATE = "PURCHASE_DATE";
+export const SAVE_ASSET = "SAVE_ASSET";
+const portfolioReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case SELECT_COIN: {
+      return {
+        ...state,
+        selectedCoin: action.payload,
+      };
+    }
+    case GET_COIN_HISTORY: {
```

Comment:
Here's where I'm trying to change each coin in the assets array, when I console log it everything looks fine but for some reason it's not working... Any ideas?